examples/simple-console/src/main/java/com/snowplowanalytics/Main.java

```diff
@@ -98,46 +102,66 @@ public static void main(String[] args) {
             .state("state")
             .country("country")
             .currency("currency")
-            .items(item) // EcommerceTransactionItem events are added to a parent EcommerceTransaction
-            .customContext(contexts)
+            .items(item) // EcommerceTransactionItem events are added to a parent EcommerceTransaction here
+            .customContext(context)
             .build();
 
-        tracker.track(ecommerceTransaction); // This will track two events
 
-        // This is an example of a custom "Unsutrcutred" event based on a schema
+        // This is an example of a custom "Unstructured" event based on a schema
+        // Unstructured events are also called "self-describing" events
+        // because of their SelfDescribingJson base
         Unstructured unstructured = Unstructured.builder()
             .eventData(new SelfDescribingJson(
                     "iglu:com.snowplowanalytics.iglu/anything-a/jsonschema/1-0-0",
                     ImmutableMap.of("foo", "bar")
             ))
-            .customContext(contexts)
+            .customContext(context)
             .build();
 
-        tracker.track(unstructured);
 
         // This is an example of a ScreenView event which will be translated into an Unstructured event
         ScreenView screenView = ScreenView.builder()
             .name("name")
             .id("id")
-            .customContext(contexts)
+            .customContext(context)
             .build();
 
-        tracker.track(screenView);
 
         // This is an example of a Timing event which will be translated into an Unstructured event
         Timing timing = Timing.builder()
             .category("category")
             .label("label")
             .variable("variable")
             .timing(10)
-            .customContext(contexts)
+            .customContext(context)
             .build();
 
+        // This is an example of a Structured event
+        Structured structured = Structured.builder()
+                .category("category")
+                .action("action")
+                .label("label")
+                .property("property")
+                .value(12.34)
+                .customContext(context)
+                .build();
+
+        tracker.track(pageViewEvent); // the .track method schedules the event for delivery to Snowplow
+        tracker.track(ecommerceTransaction); // This will track two events
+        tracker.track(unstructured);
+        tracker.track(screenView);
         tracker.track(timing);
+        tracker.track(structured);
+
+        Thread.sleep(1000);
```

Comment:
Why sleep for a second? Shouldn't `tracker.close()` handle sending all the events before closing out?

Reply:
That's true! While I was debugging I had the Emitter printing things out when it sent, so I could check the batching was working. But now it's just "are there 7 events at the end" the sleep isn't necessary.